CHANGES.md

```diff
@@ -1,3 +1,10 @@
+1.8.7
```

Comment:
I am not sure, but my understanding was that last component of version is only for fixes, but this is a new feature. So I think it would make more sense to call this version 1.9.0. But I don't mind leaving 1.8.7 actually. Up to you.

Reply:
Citing @neongreen:

```
a.b.c.d

a = ALL-NEW AWESOME COOL, i.e. number useful for marketing basically
b = actually meaningful “this version definitely breaks API” number
c = “this version adds features but doesn't break API” number
d = “I fixed haddocks”, “I fixed compilation with GHC 7.6.2”, etc
```

Reply:
Oh, I see. @ChShersh , what are your thought?

Reply:
@vrom911 I think `1.8.7` is okay since it doesn't break API.

Reply:
If we are using this convention for `log-warper`, then alright, `1.8.7` makes sense.